sdks/java/io/cassandra/src/test/java/org/apache/beam/sdk/io/cassandra/CassandraIOTest.java

```diff
@@ -110,15 +112,40 @@ public Void apply(Iterable<KV<String, Long>> input) {
     pipeline.run();
   }
 
+  @Test
+  public void testWrite() throws  Exception {
+    FakeCassandraService service = new FakeCassandraService();
+
+    ArrayList<Scientist> data = new ArrayList<>();
+    for (int i = 0; i < 1000; i++) {
+      Scientist scientist = new Scientist();
+      scientist.id = i;
+      scientist.name = "Name " + i;
+      data.add(scientist);
+    }
+
+    pipeline
+        .apply(Create.of(data))
+        .apply(CassandraIO.<Scientist>write().withCassandraService(service)
+            .withKeyspace("beam")
+            .withEntity(Scientist.class));
+    pipeline.run();
+
+    assertEquals(FakeCassandraService.TABLE.size(), 1000);
```

Comment:
Why do you need to access the static variable, rather than doing something like `service.getTable()` ?